core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/ClusterOperationService.java

```diff
@@ -550,7 +550,7 @@ public FlowIdentifier restartClusterServices(Stack stack) {
                     "Stack '%s' has state '%s' which is not supported for restarting the cluster services.", stack.getName(), stack.getStatus()
             ));
         }
-        return flowManager.triggerClusterServicesRestart(stack.getId());
+        return flowManager.triggerClusterServicesRestart(stack.getId(), true);
```

Comment:
I only see `true` constant in code and only see `false` constant in the test. It makes me think the parameter is obsolete and the test does not cover the code. Is it so?